File: C  programs/Problems on Pattern Printing/Rows & Columns/Pattern1.c
/* Write a program which accept the number of rows and number of columns from user and Display below pattern.

Input:  iRow=4   iCol=3
Output :  
          *  *  *
          *  *  *
	      *  *  *
          *  *  *

*/

#include<stdio.h>

void Pattern(int iRow,int iCol)
{
	int i=0;
	int j=0;

	if(iRow<0)
	{
	   iRow=-iRow;
	}

	if(iCol<0)
	{
	   iCol=-iCol;
	}

	for(i=1;i<=iRow;i++)
	{  
       for(j=1;j<=iCol;j++)
	   {
		   printf("*");
	   }
	   printf("\n");
    }
}

int main()
{
  int iValue1=0;
  int iValue2=0;

  printf("Enter number of rows: ");
  scanf("%d",&iValue1);

  printf("Enter number of Columns: ");
  scanf("%d",&iValue2);

  Pattern(iValue1,iValue2);

  return 0;
}  	  

File: Java programs/Problems on Pattern Printing/Print square and number pattern/Pattern1.java
/* Write a java program which Accept the number of rows and number of columns from user and Display below pattern.

Input:  iRow=4   iCol=5
Output : 4 4 4 4 4
         3 3 3 3 3
         2 2 2 2 2
         1 1 1 1 1		

*/
		  
import java.util.*;

class Pattern
{
	public String str;
    public void Pattern(int iRow,int iCol)
    {
	   int i=0;
	   int j=0;
	   
	   for(i=iRow;i>=1;i--)
	   {  
           for(j=1;j<=iCol;j++)
	       {
		      System.out.print(i+"\t");
		   }
		   System.out.println();
	   }
	}
}

class Lba364
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);

		int iValue1=0,iValue2=0;

        System.out.println("Enter number of Rows ");
        iValue1=sobj.nextInt();

		 System.out.println("Enter number of Columns");
		iValue2=sobj.nextInt();

		Pattern pobj=new Pattern();
		
        pobj.Pattern(iValue1,iValue2);
	}	
}  	  